pkg/db/drivers/etcd/etcd.go

```diff
@@ -260,6 +260,17 @@ func (c *Client) FindFileShareValue(k string, p *model.FileShareSpec) string {
 }
 
 func (c *Client) CreateFileShareAcl(ctx *c.Context, fshare *model.FileShareAclSpec) (*model.FileShareAclSpec, error) {
+	acls, err := c.ListFileSharesAcl(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, acl := range acls {
+		if acl.AccessTo == fshare.AccessTo {
+			log.Error("acl already exists for this ip: %s",acl.AccessTo)
+			return nil, fmt.Errorf("acl already exists for this ip: %s", acl.AccessTo)
```

Comment:
May be we can guide the user with proper message a little bit. For example:
"acl already exists for this IP. If you want to set new acl, first delete the existing one"

Reply:
That will be update Acl API, we are not supporting right? Will update the document